src/SprykerEco/Yves/Payone/PayoneFactory.php

```diff
@@ -55,23 +68,23 @@ class PayoneFactory extends AbstractFactory
     /**
      * @return \SprykerEco\Yves\Payone\Form\PrePaymentForm
      */
-    public function createPrePaymentForm()
+    public function createPrePaymentForm(): AbstractPayoneSubForm
```

Comment:
* Change a return typehint in the document block.
* Check other places.

Reply:
++